compiler/tools/fpp-to-cpp/test/component/test/check-cpp

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# ----------------------------------------------------------------------
+# Compile ref C++ files, to check them for validity
+# ----------------------------------------------------------------------
+
+fprime_gcc=../../../../../scripts/fprime-gcc
+export FPRIME_GCC_FLAGS="-I../../fprime"
+
+dirn=`dirname $PWD`
+echo "generating C++"
+fpp-to-cpp -d ../../fprime/Fw -p `dirname $dirn`/fprime `cat deps.txt`
+for dir in Cmd Log Prm Time Tlm
+do
+  mv ../../fprime/Fw/$dir*.hpp ../../fprime/Fw/$dir*.cpp ../../fprime/Fw/$dir
+done
+mv ../../fprime/Fw/Param*.hpp ../../fprime/Fw/Param*.cpp ../../fprime/Fw/Prm
+
+guards=`grep '#if FW_' *ComponentAc.hpp *ComponentAc.cpp | cut -f 2 -d ' ' | sort -u | sed 's/^/-D/g'`
+guards_on=`echo $guards | sed 's/ /\n/g' | sed 's/$/=1/g'`
+guards_off=`echo $guards | sed 's/ /\n/g' | sed 's/$/=0/g'`
+guards_off_serial_on=`echo $guards_off | sed 's/0$/1/g'`
+
+for file in `find . -name '*Ac.cpp'`
+do
+  echo "compiling $file with default guards"
+  $fprime_gcc -I../../.. -c $file -Wno-gnu-zero-variadic-macro-arguments -Wno-unused-parameter
+
+  g=`echo $guards_on`
+  echo "compiling $file with $g"
+  $fprime_gcc -I../../.. -c $file $g -Wno-gnu-zero-variadic-macro-arguments -Wno-unused-parameter
+
+#  g=`echo $guards_off`
```

Comment:
Why is this code commented out? Can we delete it?

Reply:
This is an old version of the script from an old test directory that I created to compile files generated by the Python Autocoder, and should have been deleted. The current version of this script is here: [compiler/tools/fpp-to-cpp/test/component/check-cpp ](https://github.com/fprime-community/fpp/pull/240/files#diff-f663fb671471e8802cc40fa486edfb4761642c0ec5ce509644c427863fae9cd6)